users/users/users.php

```diff
@@ -37,14 +41,158 @@ class UsersApiResourceUsers extends ApiResource
 	 */
 	public function delete()
 	{
-		$this->plugin->setResponse( JText::_( 'PLG_API_USERS_IN_DELETE_FUNCTION_MESSAGE' ));
+		$app              = JFactory::getApplication();
+		$db 			  = JFactory::getDbo();
+		$data             = array();
+		$eobj             = new stdClass;
+		$eobj->status     = false;
+		$eobj->id         = 0;
+
+		// Get values paased by put in the url
+		$data = $app->input->getArray(array());
+
+		// Check username or user_id to edit the details of user
+		if (isset($data['username']) || isset($data['user_id']))
+		{
+			if (!$data['user_id'] && isset($data['username']))
```

Comment:
Change this to just 
```
if (!$data['user_id'] && $data['username']) 
```

isset not needed since you have already put that check on the top